src/Symfony/Component/Console/Application.php

```diff
@@ -181,14 +181,25 @@ public function run(InputInterface $input = null, OutputInterface $output = null
         return $exitCode;
     }
 
+    /**
+     * Checks if the current application has the parameter in the default option.
+     */
+    final protected function hasDefaultOption(string $option): bool
+    {
+        $defaultDefinition = $this->getDefaultInputDefinition();
+
+        return $defaultDefinition->hasOption($option);
```

Comment:
Could be inlined to `return $this->getDefaultInputDefinition()->hasOption($option)`.

Reply:
I would also make it private.